osquery/tables/applications/windows/carbon_black.cpp

```diff
@@ -26,17 +28,18 @@ namespace tables {
 
 // Carbon Black registry path
 #define kCbRegLoc "SOFTWARE\\CarbonBlack\\config"
-// Path to Carbon Black direcotry
-#define kCbDir "C:\\Windows\\CarbonBlack\\"
 
 void getQueue(Row& r) {
+  char windowsPath[MAX_PATH];
+  GetWindowsDirectory(windowsPath, MAX_PATH);
```

Comment:
I think this is OK. Maybe we need a better abstraction for this, returned as a `fs::path` so you can then:
```cpp
fs::path root = getWindowsRoot();
root /= "CarbonBlack"
```

/cc @yying @PoppySeedPlehzr